From cd1962b7f9c14bc67e22c07cb9a9cd3a4a119506 Mon Sep 17 00:00:00 2001
From: Gaetan Nadon <memsize-XzQKRVe1yT0V+D8aMU/kSg@public.gmane.org>
Date: Sat, 28 Nov 2009 21:32:47 -0500
Subject: [PATCH] configure.ac: error while checking for XDMXCONFIG_DEP

Introduced in commit 9998105a387e0294054502331a56e1e020cd93e4
The replacement third parameters to PKG_CHECK_MODULES([DMXMODULES]
was not quoted.

Signed-off-by: Gaetan Nadon <memsize-XzQKRVe1yT0V+D8aMU/kSg@public.gmane.org>
Reviewed-by: Dan Nicholson <dbn.lists-Re5JQEeQqe8AvxtiuMwx3w@public.gmane.org>
Tested-by: Julien Cristau <jcristau at debian.org>
---
 configure.ac |   14 +++++++++-----
 1 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8500db4..1038734 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1808,11 +1808,15 @@ AM_CONDITIONAL(XQUARTZ_SPARKLE, [test "x$XQUARTZ_SPARKLE" != "xno"])
 AM_CONDITIONAL(STANDALONE_XPBPROXY, [test "x$STANDALONE_XPBPROXY" = xyes])
 
 dnl DMX DDX
-PKG_CHECK_MODULES([DMXMODULES],
-    [xmuu $LIBXEXT x11 xrender xfixes xfont $LIBXI $DMXPROTO xau $XDMCP_MODULES],
-    PKG_CHECK_MODULES([XDMXCONFIG_DEP], [xaw7 xmu xt xpm x11], [have_dmx=yes],
-                      [have_dmx=no]),
-    [have_dmx=no])
+PKG_CHECK_MODULES(
+	[DMXMODULES],
+	[xmuu $LIBXEXT x11 xrender xfixes xfont $LIBXI $DMXPROTO xau $XDMCP_MODULES],
+	[PKG_CHECK_MODULES(
+		[XDMXCONFIG_DEP],
+		[xaw7 xmu xt xpm x11],
+		[have_dmx=yes],
+		[have_dmx=no])],
+	[have_dmx=no])
 AC_MSG_CHECKING([whether to build Xdmx DDX])
 if test "x$DMX" = xauto; then
 	DMX="$have_dmx"
-- 
1.6.0.4

